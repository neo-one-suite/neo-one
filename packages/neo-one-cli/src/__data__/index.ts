export * from './getClients';
export * from './getContracts';
export * from './buildTests';
export * from './generateTests';
