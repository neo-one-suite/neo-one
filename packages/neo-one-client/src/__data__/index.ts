export * from './keys';
export * from './transactions';
