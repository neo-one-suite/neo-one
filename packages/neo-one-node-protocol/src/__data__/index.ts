export { context } from './context';
export { keys } from './keys';
