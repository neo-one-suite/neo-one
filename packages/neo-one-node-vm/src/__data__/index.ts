import * as assets from './assets';
import * as transactions from './transactions';
import * as testUtils from './utils';

export * from './createBlockchain';
export * from './keys';

export { assets, testUtils, transactions };
