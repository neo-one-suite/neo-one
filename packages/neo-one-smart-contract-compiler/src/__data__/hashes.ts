export const hashes = {
  OLD_NEO_ASSET_HASH: '0xc56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b',
  OLD_GAS_ASSET_HASH: '0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7',
};
