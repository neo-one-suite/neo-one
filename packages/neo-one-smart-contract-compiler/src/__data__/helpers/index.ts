export * from './compile';
export * from './execute';
export * from './extractors';
export * from './startNode';
export * from './utils';
