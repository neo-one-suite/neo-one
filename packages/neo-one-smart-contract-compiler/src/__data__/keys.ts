import { common } from '@neo-one/client-core';

export const keys = [
  {
    name: 'first',
    address: 'ALq7AWrhAueN6mJNqk6FHJjnsEoPRytLdW',
    privateKey: common.bufferToPrivateKey(
      Buffer.from('7d128a6d096f0c14c3a25a2b0c41cf79661bfcb4a8cc95aaaea28bde4d732344', 'hex'),
    ),
    publicKey: common.bufferToECPoint(
      Buffer.from('02028a99826edc0c97d18e22b6932373d908d323aa7f92656a77ec26e8861699ef', 'hex'),
    ),
    wif: 'L1QqQJnpBwbsPGAuutuzPTac8piqvbR1HRjrY5qHup48TBCBFe4g',
    password: 'city of zion',
    encryptedWIF: '6PYLHmDf6AjF4AsVtosmxHuPYeuyJL3SLuw7J1U8i7HxKAnYNsp61HYRfF',
    scriptHash: common.bufferToUInt160(Buffer.from('3775292229eccdf904f16fff8e83e7cffdc0f0ce', 'hex')),
  },
  {
    name: 'second',
    address: 'ALfnhLg7rUyL6Jr98bzzoxz5J7m64fbR4s',
    privateKey: common.bufferToPrivateKey(
      Buffer.from('9ab7e154840daca3a2efadaf0df93cd3a5b51768c632f5433f86909d9b994a69', 'hex'),
    ),
    publicKey: common.bufferToECPoint(
      Buffer.from('031d8e1630ce640966967bc6d95223d21f44304133003140c3b52004dc981349c9', 'hex'),
    ),
    wif: 'L2QTooFoDFyRFTxmtiVHt5CfsXfVnexdbENGDkkrrgTTryiLsPMG',
    password: '我的密码',
    encryptedWIF: '6PYWVp3xfgvnuNKP7ZavSViYvvim2zuzx9Q33vuWZr8aURiKeJ6Zm7BfPQ',
    scriptHash: common.bufferToUInt160(Buffer.from('35b20010db73bf86371075ddfba4e6596f1ff35d', 'hex')),
  },
  {
    name: 'third',
    address: 'AVf4UGKevVrMR1j3UkPsuoYKSC4ocoAkKx',
    privateKey: common.bufferToPrivateKey(
      Buffer.from('3edee7036b8fd9cef91de47386b191dd76db2888a553e7736bb02808932a915b', 'hex'),
    ),
    publicKey: common.bufferToECPoint(
      Buffer.from('02232ce8d2e2063dce0451131851d47421bfc4fc1da4db116fca5302c0756462fa', 'hex'),
    ),
    wif: 'KyKvWLZsNwBJx5j9nurHYRwhYfdQUu9tTEDsLCUHDbYBL8cHxMiG',
    password: 'MyL33tP@33w0rd',
    encryptedWIF: '6PYNoc1EG5J38MTqGN9Anphfdd6UwbS4cpFCzHhrkSKBBbV1qkbJJZQnkn',
    scriptHash: common.bufferToUInt160(Buffer.from('9847e26135152874355e324afd5cc99f002acb33', 'hex')),
  },
];
