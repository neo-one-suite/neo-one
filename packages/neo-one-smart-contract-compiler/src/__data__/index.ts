import * as helpers from './helpers';

export { helpers };
export * from './keys';
