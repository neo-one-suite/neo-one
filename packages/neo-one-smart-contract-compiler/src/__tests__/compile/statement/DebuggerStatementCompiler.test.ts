import { helpers } from '../../../__data__';

describe('DebuggerStatementCompiler', () => {
  test('simple', async () => {
    await helpers.compileString(
      `
      let result = 0;

      debugger;

      if (result !== 0) {
        throw 'Failure';
      }
    `,
      {
        type: 'error',
      },
    );
  });
});
