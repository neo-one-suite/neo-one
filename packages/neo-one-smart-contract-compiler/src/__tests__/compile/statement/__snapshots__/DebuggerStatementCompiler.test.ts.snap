// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DebuggerStatementCompiler simple 1`] = `
"snippetCode.ts (4,7): Unsupported syntax

      2 |       let result = 0;
      3 | 
    > 4 |       debugger;
        |       ^
      5 | 
      6 |       if (result !== 0) {
      7 |         throw 'Failure';
"
`;
